react-common/styles/profile/profile.css

```diff
@@ -2,6 +2,8 @@
     display: flex;
     flex-direction: row;
     height: 100%;
+
+    --background-color: #fef3e0;
```

Comment:
should this be the var(--background-color)?

Reply:
this is defining the variable! hopefully in the future i can switch over to less and just reuse those vars